priv/repo/migrations/20250106164510_rail_line_add_unknown.exs

```diff
@@ -0,0 +1,7 @@
+defmodule Orbit.Repo.Migrations.CertificationRailLineUnknown do
+  use Ecto.Migration
+
+  def change do
+    execute "ALTER TYPE rail_line ADD VALUE 'none';"
```

Comment:
Oh, it just occurred to me that the associated rail_line column in `certifications` needs a `NOT NULL` on it now. I'll fix tomorrow

Reply:
fixed